cli-plugins/apps/reference/workload-tail.hbs.md

```diff
@@ -82,7 +84,9 @@ pet-clinic-00004-deployment-6445565f7b-ts8l5[workload] 2022-06-14 16:28:53.231
 
 ### <a id="tail-since"></a> `--since`
 
-Sets the time duration to start reading logs from, this is set in seconds (`s`), minutes(`m`) or hours (`h`) in the format `0h0m0s`. You do not need to indicate a `0` duration, for example, 1 hour, 0 minutes and 1 seconds is `1h1s`. The default value is 1 second `1s`
+Sets the time duration to start reading logs from, this is set in seconds (`s`), minutes(`m`) or 
```

Comment:
Sets the time duration to start reading logs from. 

The unit value can be set in seconds (`s`), minutes(`m`) or hours (`h`) in the format `0h0m0s`. You do not need to indicate a `0` duration for a unit that is not being expressed (e.g. 1 hour,
0 minutes and 1 seconds should be expressed at `1h1s`). 

The default value is 1 second `1s`.

Reply:
changed